Parser.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import csv
+import json
+import xml.dom.minidom
+from datetime import datetime
+from datetime import timedelta
+
+
+class Parser:
+    def __init__(self, filename):
+        self.filename = filename
+        self.localTransactions = []
```

Comment:
Would be better if we created a new model called Transaction that we could assign the data to in here rather than trying to create multiple objects of the same structure